src/lib/web-component/log-view/log-view-element.js

```diff
@@ -27,10 +29,19 @@ export class LogView extends HTMLElement {
 
   setLogFilter(filterFunc) {
     this.isFilterMatched = filterFunc;
+    let filteredLogs = 0;
 
     for (const row of this.tableBody.rows) {
       row.hidden = !this.isFilterMatched(row._log);
+
+      if (!row.hidden) {
+        filteredLogs++;
+      }
     }
+
+    this.dispatchEvent(
+      new CustomEvent('logcounter', { detail: { filteredLogs } })
```

Comment:
`logcountchange` would be a better event name than "logcounter".

Reply:
yes, it is. Updated to the event name to `logcountchange`. :-)